apps/web/docker/Dockerfile

```diff
@@ -17,38 +18,37 @@ RUN --mount=type=cache,id=pnpm,target=/pnpm/store pnpm install \
   --frozen-lockfile \
   --filter "@latitude-data/web..."
 
-# PRODUCTION stage
-# ------------------------------------------------------
-FROM installer AS builder
-
 RUN --mount=type=cache,id=pnpm,target=/pnpm/store \
   BUILDING_CONTAINER=true \
   pnpm turbo build --filter='@latitude-data/web...'
 
-FROM base AS runner
-
+# PRODUCTION
+FROM base AS production
 WORKDIR /app-prod
 
-# Don't run production as root
+# Public files doesn't have the unpriviledged user
+COPY --from=builder /app/public ./public
+
+# Unpriviledged user
 RUN addgroup --system --gid 1001 nodejs
 RUN adduser --system --uid 1001 nextjs
 USER nextjs
 
 COPY --from=installer /app/apps/web/next.config.mjs .
 COPY --from=installer /app/apps/web/package.json .
-COPY --from=installer /app/node_modules ./node_modules
-COPY --from=installer /app/apps/web/package.json ./apps/web/package.json
```

Comment:
Double check the 2 lines are not necessary. Not sure the way pnpm works we at least need the root `app/node_modules`